frontend/src/components/Review/Review.tsx

```diff
@@ -212,15 +235,42 @@ const ReviewComponent = ({
     );
   };
 
+  const bedroomsPriceLabel = () => {
+    return (
+      <Grid item className={bedroomsPrice} style={isMobile ? { width: '100%' } : {}}>
+        {bedrooms > 0 && (
+          <div
+            className={bedroomsPrice}
+            style={isMobile ? { marginLeft: '0' } : { marginLeft: '30px' }}
+          >
+            <BedIcon className={bedPriceIcon} />
+            <Typography className={bedroomsPriceText}>
+              {bedrooms} {bedrooms == 1 ? 'Bedroom' : 'Bedrooms'}
```

Comment:
There is a warning that '===' should be used instead of '=='. In this case, it is fine because bedrooms is a number, but you may want to change it to == to resolve the warning and make the console cleaner.